README.md

```diff
@@ -23,6 +24,10 @@ If you're brand new to feature flagging, considering reviewing the [OpenFeature
   - [Environment Variable](#environment-variable)
   - [FlagD](#flagd)
     - [Flag Configuration in FlagD](#flag-configuration-in-flagd)
+      - [State](#state)
```

Comment:
I intentionally set the max depth to 4 in the markdown toc rules. If we want these here just remove the max depth option.

Reply:
https://github.com/open-feature/playground/pull/61#discussion_r987217776